.changeset/selfish-penguins-double.md

```
@@ -0,0 +1,5 @@
+---
+"@spear-ai/tailwind-config": patch
+---
+
+Removed "type": "module" from the package.json to allow for proper module resolution.
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)